### PR TITLE
remove broken, opinionated defaults

### PR DESCRIPTION
**Pull request by ghost**
These settings are annoying

If you still want a default qt theme, I propose adding `qt5ct` and `qt6ct` to `config/common-packages.yml` and setting `QT_QPA_PLATFORMTHEME=qt5ct` instead of hardcoding `QT_STYLE_OVERRIDE` and `GTK_THEME` in `/etc/environment`. For Hyprland if you want these variables set have it be in the default Hyprland config like this so it's not hidden in `/etc/environment`:

```
env = GTK_THEME,Adwaita:dark
env = QT_STYLE_OVERRIDE,adwaita-dark
env = WLR_NO_HARDWARE_CURSORS,1
```

### CODE REVIEW

**Comment by RoyalOughtness** [comment]
Can you clarify what exactly is "broken" here?

**Comment by RoyalOughtness** [comment]
No response after 1 week, closing.

**Comment by ghost** [comment]
I apologize, I was geeking when I made this PR and I'm addicted to having troll-like behavior (the title wasn't sincere, I'm sorry, and this should've been an issue). I also didn't realize changes to /etc were layered automatically so I could just edit the file and have it persist, and having these defaults set in /etc/environment caught me off guard, I think this is non-standard and I found out about it when my QT_QPA_PLATFORMTHEME=qt5ct wasn't working.

Since I actually do have an interest in having this change I'll actually explain: The reasoning was that these envars were destroying my qt apps by applying broken theming. On QT apps, particularly dolphin which is one of the only qt apps I use, icons were missing, the theming had fucked up contrast that made it difficult to read, and I couldn't even drag files or right click them because of the missing icons. I also assume most of the WM community is using qt5/qt6ct. Applying GTK_THEME like this is bad as well, I'm pretty sure the GTK devs specifically say not to set that envar to apply themes.

The main objective complaint I had was with applying WLR_NO_HARDWARE_CURSORS=1, this is a workaround and hardware cursors shouldn't be forced off like this, and the people who experience issues with hardware cursors on the atomic api may prefer to set WLR_DRM_NO_ATOMIC=1 instead and keep hardware cursors.

I think the most relevant thing here is asking who these images are for. I switched to ostree from arch+hyprland, because after years of running an incredibly janky setup, an update prevented me from even getting to a TTY. I know for a fact this wasn't an issue with arch itself but was 100% caused by some form of configuration drift because when I booted a brand new archiso to try and fix it I had no problems there.

So consider the demographic of someone who is going to use an rpm-ostree distro and rebase to an image shipping a non-DE wlroots compositor where you're expected to configure things yourself. They're likely to be open-minded tech enthusiasts who aren't afraid to google things when they have an issue and apply a workaround. Why should these settings be applied like this? No other distro that I know of does something like this. The only way I found out about /etc/environment is because when I opened qt5ct it said the theme was set to adwaita-qt, where I then went to check `ls /etc/environment.d` then checked /etc/environment. Imagine if you're not part of the minority of people having bugs with hardwarecursors and all you know is your cursor is all weird. How would you know this variable was changed in this system file that's normally for manual user overrides unless you were familiar with these specific variables and guessed that the image shipped with these changes?

TLDR: It would be better UX to just have people who need workarounds that have negative consequences to apply them by themselves and this isn't an unrealistic expectation for what will likely be the demographic for this image.

Anyways, sorry, for writing an essay but I felt bad about not elaborating originally and thoughts just kept coming to my head. "If I had more time I would've written a shorter letter or whatever" —someone

**Comment by PaniniLinguini** [comment]
https://github.com/wayblueorg/wayblue/blob/live/config/files/usr/etc/profile.d/settheming.sh No way you put this shit here too

**Comment by RoyalOughtness** [comment]
> I'm sorry, and this should've been an issue

No problem, thanks for your sincere apology. The concerns you raise are legitimate.

> didn't realize changes to /etc were layered automatically so I could just edit the file and have it persist

to be precise, they're not layered per se but untouched by the system if modified

> I also assume most of the WM community is using qt5/qt6ct. Applying GTK_THEME like this is bad as well, I'm pretty sure the GTK devs specifically say not to set that envar to apply themes.

That's a fair point. Can you submit a new PR that keeps the current default theme but via qt5/6ct so it can be overriden more easily?

> someone who is going to use an rpm-ostree distro and rebase to an image shipping a non-DE wlroots compositor where you're expected to configure things yourself. They're likely to be open-minded tech enthusiasts who aren't afraid to google things when they have an issue and apply a workaround. 

The point of these images is to have a reasonably aesthetically pleasing and functional WM-based system ootb. It includes the audience you describe, but they're a subset, since they can of course override the themes.

> asking who these images are for.

People who want a sane defaults WM image that works ootb, and those who want to use it as a base. Both.

> No other distro 

To be precise, the distro here is Fedora :)

Wayblue is not a distro

> people who experience issues with hardware cursors on the atomic api may prefer to set WLR_DRM_NO_ATOMIC=1 instead and keep hardware cursors.

nothing is forced and everything is overridable.

all the hardware I tested with an external monitor failed to display a cursor on the external monitor without that variable set. If you have a recommendation for this so it works ootb for most hardware, please let me know

**Comment by RoyalOughtness** [comment]
@PaniniLinguini you have been banned for 1 week. please review the code of conduct
